### ISSUE TITLE
MakeMKV fails - but only when udev runs script

### ISSUE THREAD

**Issue by ChimneySwift** [automatic-ripping-machine/automatic-ripping-machine#250]
### Description
MakeMKV reports no discs and says "Failed to open disc" when ARM is run automatically by udev:

```
MSG:1005,0,1,"MakeMKV v1.12.3 linux(x64-release) started","%1 started","MakeMKV v1.12.3 linux(x64-release)"
DRV:0,256,999,0,"","",""
DRV:1,256,999,0,"","",""
DRV:2,256,999,0,"","",""
DRV:3,256,999,0,"","",""
DRV:4,256,999,0,"","",""
DRV:5,256,999,0,"","",""
DRV:6,256,999,0,"","",""
DRV:7,256,999,0,"","",""
DRV:8,256,999,0,"","",""
DRV:9,256,999,0,"","",""
DRV:10,256,999,0,"","",""
DRV:11,256,999,0,"","",""
DRV:12,256,999,0,"","",""
DRV:13,256,999,0,"","",""
DRV:14,256,999,0,"","",""
DRV:15,256,999,0,"","",""
MSG:5010,0,0,"Failed to open disc","Failed to open disc"
```

However, when I run ARM manually from the terminal (`/opt/arm/scripts/arm_wrapper.sh '/dev/sr0'``), MakeMKV runs successfully and ARM finishes.

It does this with every disc I've tried.

When I run it from the terminal, I do get this error, not sure if this is related:
```
mount: /mnt/dev/sr0: WARNING: device is write-protected, mounting read-only
```

### Environment
Ubuntu 18.04.1 LTS
2.0.1-stable

### Log file
[THE_HITMAN_S_BODYGUARD.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/2408489/THE_HITMAN_S_BODYGUARD.log)

**Comment by ChimneySwift** [comment]
It looks like someone else has had the same issue: https://www.makemkv.com/forum/viewtopic.php?t=15873

I might try his hacky suggestion to get this work until there's a proper fix.

**Comment by muckngrind4** [comment]
Hi @ChimneySwift.  When you run it manually are you the 'arm' user?  If not, can you try it as 'arm'?  MakeMKV is reporting and exit code (10), but it's really hard to find documentation on MakeMKV exit codes...

**Comment by ChimneySwift** [comment]
I figured out the issue... It was basically entirely my fault.

Udev wasn't working properly to start, so I was messing with the wrapper script and I changed the way it runs ARM to use `&` instead of `at now`, I still don't know why MakeMKV fails if Udev runs it with `&`, which is probably why it took me so long to figure out that was my issue. 

Restoring arm_wrapper.sh solved my problem. Sorry for the confusion.
